src/common/function/FunctionManager.cpp

```diff
@@ -2092,29 +2092,15 @@ FunctionManager::FunctionManager() {
   {
     auto &attr = functions_["coalesce"];
     attr.minArity_ = 1;
-    attr.maxArity_ = 1;
+    attr.maxArity_ = INT64_MAX;
```

Comment:
emmm, Seems it's not unachievable in the current FunctionManager. Plz have a look at the function signature `    {"coalesce",
     {
         TypeSignature({Value::Type::LIST}, Value::Type::__EMPTY__),
     }},`

Reply:
Sorry, I'm wrong. Add `coalese` to `variadicFunReturnType_` is ok.